### PR TITLE
PE-8498 Manage System Interfaces

### PR DESCRIPTION
**Pull request by onwsk8r**
Use the `maas_interface_physical` resource:

```hcl
resource "maas_interface_physical" "myserver_eth0" {
  system_id   = maas_instance.myserver.system_id
  name        = "eth0"
  mac_address = "01:23:45:67:89:0A"
  tags        = ["foo", "bar"]
  vlan        = "my_vlan"
  mtu         = 1500
  accept_ra   = true
  autoconf    = true
}
```

### CODE REVIEW

**Review by liskl — COMMENTED** [review]
the code itself looks good, and tests pass as well as compiles, i'm not 100% following all the code flow, but my sampling all looks good, he provider doesn't have much in the way of tests yet because we don't have infra allocated to test the provider
```
PASS
ok  	github.com/roblox/terraform-provider-maas	2.113s
?   	github.com/roblox/terraform-provider-maas/internal/bridge	[no test files]
PASS
ok  	github.com/roblox/terraform-provider-maas/internal/provider	1.335s
PASS
ok  	github.com/roblox/terraform-provider-maas/internal/tfschema	1.334s
?   	github.com/roblox/terraform-provider-maas/pkg/api	[no test files]
?   	github.com/roblox/terraform-provider-maas/pkg/api/params	[no test files]
PASS
ok  	github.com/roblox/terraform-provider-maas/pkg/gmaw	1.336s
PASS
ok  	github.com/roblox/terraform-provider-maas/pkg/maas	1.230s
PASS
ok  	github.com/roblox/terraform-provider-maas/pkg/maas/entity	1.247s
PASS
ok  	github.com/roblox/terraform-provider-maas/pkg/maas/entity/subnet	1.240s
?   	github.com/roblox/terraform-provider-maas/test/helper	[no test files]
```

File: README.md [hunk 1]
```diff
@@ -37,115 +42,164 @@ This provider is only able to deploy and release nodes already registered and co
...
+  count = 3
 }
 ```
 
+#### maas_interface_physical
+
+Configures a physical interface on a system, where a system is anything with a system ID.
+
+```hcl
+resource "maas_interface_physical" "myserver_eth0" {
```
Comment (robloxrob) [inline README.md:84]: Thank you for adding this to the readme

File: pkg/maas/entity/subnet/statistics.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package subnet
+
+import "net"
+
+type Statistics struct {
```
Comment (robloxrob) [inline pkg/maas/entity/subnet/statistics.go:5]: docs as needed

**Review by robloxrob — CHANGES_REQUESTED** [review]
Docs as needed. Amazing work!!!!!!!!!!!!!! 🥇

**Pull request merged**